src/core/database/query/lists.ts

```diff
@@ -66,3 +66,10 @@ export const defaultSorter =
           .orderBy(sortValSecuredProp, order)
       : q.with('*').orderBy(sortValBaseNodeProp, order);
   };
+
+export const whereNotDeletedInChangeset = (changeset?: ID) => (query: Query) =>
+  changeset
+    ? query.raw(
+        `WHERE NOT (node)<-[:changeset { active: true, deleting: true }]-(:Changeset { id: '${changeset}' })`
+      )
```

Comment:
Please bind values. I'm pretty sure it's not just safer, it helps with performance as well (since a hard coded value would make each query it would need to be parsed every time).
```suggestion
        'WHERE NOT (node)<-[:changeset { active: true, deleting: true }]-(:Changeset { id: $changesetId })',
        { changesetId: changeset }
      )
```